napari/layers/image/_image_slice.py

```diff
@@ -1,54 +1,124 @@
-from typing import Optional
+"""ImageSlice class.
+"""
+import logging
+
+import numpy as np
 
 from ...types import ArrayLike, ImageConverter
+from ...utils.chunk import ChunkKey, ChunkRequest, chunk_loader
 from ._image_view import ImageView
 
+LOGGER = logging.getLogger('ChunkLoader')
+
 
 class ImageSlice:
     """The slice of the image that we are currently viewing.
 
-    Right now this just holds the image and its thumbnail, however future async
-    and multiscale-async changes will likely grow this class a lot.
+    Holds the image and its thumbnail and has load_chunk() and chunk_loaded() methods.
 
     Parameters
     ----------
     view_image : ArrayLike
-        The default image for the time and its thumbail.
-    image_converter : ImageConverter, optional
+        The initial image used as the image and the thumbnail source.
+    image_converter : ImageConverter
         ImageView uses this to convert from raw to viewable.
+    rgb : bool
+        Is the image in RGB or RGBA format.
 
     Attributes
     ----------
     image : ImageView
         The main image for this slice.
-
     thumbnail : ImageView
-        The smaller thumbnail image for this slice.
+        The source image used to compute the smaller thumbnail image.
+    rgb : bool
+        Is the image in RGB or RGBA format.
+    current_key
+        The ChunkKey we are currently showing or which is loading.
+    loaded : bool
+        Has the data for this slice been loaded yet.
+    """
 
-    Examples
-    --------
-    Create with some default image:
+    def __init__(
+        self,
+        view_image: ArrayLike,
+        image_converter: ImageConverter,
+        rgb: bool = False,
+    ):
+        LOGGER.info("ImageSlice.__init__")
```

Comment:
would like to see many of these LOGGER calls reduced to `LOGGER.debug`.  seeing that an object is instantiated, for instance, definitely seems more appropriate for debug